shuffle/cli/tutorials/HelloNFT.md

```diff
@@ -106,7 +106,7 @@ Let's try calling the `mint_nft` script function within the console.
 
 > let txn = await helpers.invokeScriptFunction(scriptFunction, typeArguments, args);
 
-> txn = await devapi.waitForTransaction(txn.hash);
+> txn = await devapi.waitForTransactionCompletion(txn.hash);
```

Comment:
In a fast follow, let's rename this method to `waitForTransaction` to have parity with the Dev API's `client.ts`:

https://github.com/diem/diem/blob/cb5ada96dd881864ffe0eee0f5dbdca12845eac8/shuffle/move/examples/main/client.ts#L30-L33